engine/test/integration_tests/dm_full_mode/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -eu
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CONFIG="$DOCKER_COMPOSE_DIR/3m3e.yaml $DOCKER_COMPOSE_DIR/dm_databases.yaml"
+TABLE_NUM=500
+
+function run() {
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
+
+	start_engine_cluster $CONFIG
+	wait_mysql_online.sh --port 3306
+	wait_mysql_online.sh --port 4000
+
+	# prepare data
+
+	run_sql 'DROP DATABASE IF EXISTS dm_full_mode'
+	run_sql 'CREATE DATABASE dm_full_mode;'
+	for i in $(seq $TABLE_NUM); do
+		run_sql "CREATE TABLE dm_full_mode.t$i(i TINYINT, j INT UNIQUE KEY);"
+		for j in $(seq 2); do
```

Comment:
Maybe more data will be better.

Reply:
Will add more large size test as well as multiple jobs later. This is only a medium size test